.github/workflows/e2e/k8s/sample-job.yml

```
@@ -22,7 +22,7 @@ spec:
         imagePullPolicy: IfNotPresent
         command: ["/bin/sh", "-c"]
         # send SIGTERM to otel-go-instrumentation once the sample app has generated data so the job completes.
-        args: ["/sample-app/main && kill -TERM $(pidof otel-go-instrumentation)"]
+        args: ["/sample-app/main && kill --verbose --timeout 1000 TERM --timeout 1000 KILL $(pidof otel-go-instrumentation)"]
```

should it not be something like

```suggestion
        args: ["/sample-app/main && kill --verbose --signal TERM --timeout 1000 KILL $(pidof otel-go-instrumentation)"]
```

---

oh I see that we are giving 1sec for the app, right?